app/src/main/java/com/sopt/now/compose/Profile/ProfileFriendItem.kt

```diff
@@ -0,0 +1,64 @@
+package com.sopt.now.compose.Profile
+import androidx.compose.foundation.Image
+import androidx.compose.ui.Modifier
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.sopt.now.compose.Friend
+import com.sopt.now.compose.R
+
+
+@Composable
+fun ProfileFriendItem(friend: Friend) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(vertical = 10.dp, horizontal = 10.dp)
+            .height(110.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ){
+        val image: Painter = painterResource(id = R.drawable.ic_person_white_24)
```

Comment:
```suggestion
        val image: Painter = painterResource(friend.profileImage)
```
로 viewmodel에 저장된 이미지를 불러오는 건 어떤가요?

Reply:
또 이미지는 아래 painter에 곧바로 들어가면 조금더 가독성이 좋을것같아요! 

val image변수를 따로 선언안해도 될것 같아요!